hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/ha/OMFailoverProxyProvider.java

```diff
@@ -68,70 +53,39 @@
  * multiple OMs to connect to. In case of OM failover, client can try
  * connecting to another OM node from the list of proxies.
  */
-public class OMFailoverProxyProvider<T> implements
-    FailoverProxyProvider<T>, Closeable {
+public class OMFailoverProxyProvider<T> extends
```

Comment:
We should probably rename this to `HadoopRpcOMFailoverProvider`

Reply:
Sounds good.  We can rename it so we'll have the `OMFailoverProxyProviderBase` extended by `HadoopRpcOMFailoverProvider` and the `GrpcOMFailoverProvider`.